client/src/app/components/Layout/Layout.jsx

```diff
@@ -99,9 +103,15 @@ export class Layout extends PureComponent {
                 )
               }
               onClick={this.handleViewTypeChange()}
-              title={`Change to ${
-                viewType === ViewType.LIST ? 'tiles' : 'list'
-              } view`}
+              title={
```

Comment:
change to:
`title={formatMessage({ id: viewType === ViewType.LIST ? 'app.layout.list-view' : 'app.layout.tile-view' })}`